packages/sdk/contractkit/src/wrappers/SortedOracles.test.ts

```diff
@@ -19,7 +17,10 @@ TEST NOTES:
 
 testWithGanache('SortedOracles Wrapper', (web3) => {
   const kit = newKitFromWeb3(web3)
-  const SortedOracles = truffleContract(SortedOraclesABI)
+  const SortedOracles = truffleContract({
+    abi: SortedOraclesABI,
+    contractName: 'SortedOracles',
+  })
```

Comment:
So the issue is that this needs more than just "abi" and "contractName"

it needs a json object like the following

```
{
  "contractName":
  "abi":
  "metadata":
  "bytecode":
  "deployedBytecode":
  "sourceMap":
  "deployedSourceMap":
  "source":
  "sourcePath":
  "ast": {
    "absolutePath":
    "exportedSymbols":
    "id":
    "nodeType":
    "nodes":
    "src":
  }
  "legacyAST":
  "compiler":
  "networks":
  "schemaVersion":
  "updatedAt":
  "devdoc":
  "userdoc":
}
```